### ISSUE TITLE
How to enter a text cell?

### ISSUE THREAD

**Issue by toddmo** [Mathics3/mathics-django#164]
I haven't been able to find on google, stack overflow, the documentation how to simply enter a cell of type text (as opposed to a formula cell). There doesn't seem to be a menu in the django interface to select the type of cell. These are useful to add headings and comments to notebooks.

Does Mathics support creating text cells in the django interface? Is there a special key combination to do this? If so, where is that explained in any of the documentation? Thanks.

**Comment by TiagoCavalcante** [comment]
I don't think this is possible now, but you can add comments like this: `(* this is a comment *)`

**Comment by rocky** [comment]
> I don't think this is possible now, but you can add comments like this: `(* this is a comment *)`

Yes it isn't possible now. There is quite a bit still to be done  boxing and formatting before we could be able to support Cells (and more generally notebooks and interfacing with Jupyter). 

We are starting to work on this now, but we have very few people working on this. If this is something you are interested in seeing done and want to dig in, feel free to do so!
